freud/environment.pyx

```diff
@@ -435,13 +422,8 @@ cdef class LocalDescriptors:
                     ('Orientations must be given to orient LocalDescriptors '
                         'with particles\' orientations'))
 
-            orientations = freud.common.convert_array(orientations, 2)
-            if orientations.shape[1] != 4:
-                raise TypeError('orientations should be an Nx4 array')
-
-            if orientations.shape[0] != points_ref.shape[0]:
-                raise ValueError(
-                    "orientations must have the same size as points_ref")
+            orientations = freud.common.convert_array(orientations,
+                shape=(points_ref.shape[0], 4)) # noqa
```

Comment:
Not sure why this is `# noqa` - is there a `flake8` complaint? I like to mark `# noqa E501` or whatever the `flake8` rule is that it breaks.